### ISSUE TITLE
The type name 'UnityWebRequest' could not be found in the namespace 'UnityEngine.Networking'

### ISSUE THREAD

**Issue by kayson91** [playgameservices/play-games-plugin-for-unity#2758]
Assets/GooglePlayGames/ISocialPlatform/PlayGamesAchievement.cs(50,17): error CS1069: The type name 'UnityWebRequest' could not be found in the namespace 'UnityEngine.Networking'. This type has been forwarded to assembly 'UnityEngine.UnityWebRequestModule, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null' Enable the built in package 'Unity Web Request' in the Package Manager window to fix this error.

**Comment by cyberangel67** [comment]
Have you tried enabling that package from the package manager?

**Comment by Kayson91** [comment]
yes and I remove after to downgrade but have other issue with old version. so I don't understand why the new version isn't working.

**Comment by cyberangel67** [comment]
downgrading doesn't always work, due to changes between versions. If it was a patch release, it should be ok. But always backup your projects before doing a downgrade.

**Comment by Kayson91** [comment]
ok i have this issue with the new version "GooglePlayGamesPlugin-0.10.04.unitypackage"

**Comment by cyberangel67** [comment]
> ok i have this issue with the new version "GooglePlayGamesPlugin-0.10.04.unitypackage"

Might pay to say which version of Unity you are using, Unity 2019.2.8 and Unity 2019.3.0b7 don't have this issue.

**Comment by Kayson91** [comment]
I use unity 2019.2.9f1

**Comment by Kayson91** [comment]
Hello team, any feedback for this issue ??

**Comment by dylanh724** [comment]
I'm actually experiencing this with another project - although unrelated, it's the same err. Did you guys ever fix this? What was the cause and solution? 

I can't even find a package named anything related to what the err asks me to search for.

**Comment by Hshihbi** [comment]
Hshihbi

**Comment by dylanh724** [comment]
> UnityWebre...

Hmm, I think you cutoff

**Comment by curry1511** [comment]
Open PackageManager, then select Built-in, enable Unity Web Request WWW

**Comment by vaibhavgpt-google** [comment]
Closing this issue as part of a cleanup of older tickets. Google Play Games v1 is now deprecated. Please upgrade to version v2.1.0 or newer: https://github.com/playgameservices/play-games-plugin-for-unity/releases/tag/v2.1.0. If this issue persists on the new v2 architecture, please open a new ticket.
